### PR TITLE
Stamina hud

### PR DESCRIPTION
**Pull request by tortellinitony**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -33,6 +33,9 @@
 		set_species(/datum/species/tarajan, icon_update=1)
 	if(jobban_isbanned(src, CLUWNEBAN) && !dna.check_mutation(CLUWNEMUT))
 		dna.add_mutation(CLUWNEMUT)
+	if(client && hud_used)
```
Comment (j-mie) [inline code/modules/mob/living/carbon/human/life.dm]: I'd rather this was a hook proc.

Comment (tortellinitony) [reply code/modules/mob/living/carbon/human/life.dm]: ok I'll squash it all into one

**Review by GuyonBroadway — APPROVED** [review]

**Comment by GuyonBroadway** [comment]
Looks fine, has also been tested Approving for merge.

**Review by ghost — CHANGES_REQUESTED** [review]
Could you add a changelog?

**Comment by GuyonBroadway** [comment]
It's a port of something that's been core to hippie for ages, not sure it needs a changelog.

**Comment by ghost** [comment]
Sorry, thought skipped my mind. No point putting in a changelog if nobody will notice the change after the rebase.

**Review by ghost — APPROVED** [review]

**Pull request merged**